src/loom-provider.ts

```diff
@@ -307,10 +310,17 @@ export class LoomProvider {
   // PRIVATE FUNCTIONS EVM CALLS
 
   private _ethAccounts() {
-    if (this.accountsAddrList.length === 0) {
+    if (this.accounts.size === 0) {
       throw Error('No account available')
     }
-    return this.accountsAddrList
+
+    const accounts = new Array()
```

Comment:
Try `return this.accounts.keys().map(key => key)` instead

Reply:
I think it not possible because `keys` returned are `Iterable`, not an array, I tried to use map before but give a typescript error with type